tensorboard/backend/event_processing/data_ingester.py

```diff
@@ -196,3 +214,19 @@ def _parse_event_files_spec(logdir_spec):
             path = os.path.realpath(os.path.expanduser(path))
         files[path] = run_name
     return files
+
+
+def _cloud_fs_supported() -> bool:
+    """Checks if the cloud filesystems are supported."""
+    for fs in _CLOUD_FILESYSTEMS:
+        try:
+            if fs not in tf.io.gfile.get_registered_schemes():
+                return False
+        except:
```

Comment:
See above re: bare "except" clauses.

Also, for this, IMO it would be a bit better to explicitly check for the availability of the symbol.  We can do that like this:

```python
get_registered_schemes = getattr(tf.io.gfile, "get_registered_schemes", None)
if get_registered_schemes is not None:
    # Use get_registered_schemes
else:
    # Fall back to `tf.io.gfile.exists`
```

Reply:
Done.